clusterloader/config/rcpod.yaml

```diff
@@ -0,0 +1,17 @@
+provider: local
```

Comment:
When I see rc* I generally think "ReplicationController", so maybe change to consumerPod for sake of clarity.